common/djangoapps/student/views.py

```diff
@@ -350,6 +351,19 @@ def _cert_info(user, course, cert_status):
         else:
             status_dict['download_url'] = cert_status['download_url']
 
+            # getting linkedin URL and then pass the params which appears
+            # on user profile
+            linked_in_url = LinkedInUrlConfiguration.current().linkedin_url
+            cert_name = '{type} Certificate for {cert_name}'.format(
```

Comment:
Please make this a unicode string and write a test case in which `course.display_name` contains unicode characters.